purchases/src/test/resources/paywalldata-sample1.json

```diff
@@ -27,7 +27,9 @@
     "es_ES": {
       "title": "Tienda",
       "call_to_action": "Comprar",
+      "call_to_action_with_multiple_intro_offers": " ",
       "offer_details_with_intro_offer": " ",
+      "offer_details_with_multiple_intro_offers": "",
       "offer_name": "{{ period }}",
       "features": [
         {
```

Comment:
Do we want to add a `"subtitle": ""` in here to test the new `EmptyStringToNullSerializer` on `subtitle`?

Reply:
I wanted to test that missing fields decode as null still instead of crashing as well